src/sections/dataset/dataset-citation/DatasetCitation.tsx

```diff
@@ -0,0 +1,94 @@
+import { Col, Row, Tooltip, Icon } from 'dataverse-design-system'
+import styles from './DatasetCitation.module.scss'
+
+import { Citation } from '../../../dataset/domain/models/Dataset'
+
+interface DatasetCitationProps {
+  citation: Citation
+}
+
+function mapAuthors(authors: string[]): string[] {
+  return authors.map((author, index) => {
+    // If this is the last author in the array, don't add a semi-colon
+    if (index === authors.length - 1) {
+      return author
+    }
+    // Otherwise, add a semi-colon after the author
+    return `${author}; `
+  })
+}
+function getCitationText(citation: Citation) {
+  return (
+    <div>
+      {mapAuthors(citation.authors)}
+      {', '}
+      {citation.creationYear}
+      {', '}
+      {'"' + citation.title + '", '}
+      <a
+        className={styles.link}
+        href={citation.persistentIdentifierUrl}
+        target="_blank"
+        rel="noopener noreferrer">
+        {citation.persistentIdentifier}
+      </a>
+      {', '}
+      {citation.publisher}
+      {', '}
+      {citation.version}
+      {citation.version === 'DRAFT' && (
+        <span>
+          {' '}
+          <Tooltip
+            placement={'top'}
+            message={
+              'DRAFT VERSION will be replaced in the citation with the selected version once the dataset has been published.'
+            }
+          />
+        </span>
+      )}
+      {citation.isDeaccessioned && (
+        <span>
+          {', '}
+          DEACCESSIONED VERSION{' '}
+          <Tooltip
+            placement={'top'}
+            message={
+              'DEACCESSIONED VERSION has been added to the citation for this version since it is no longer available.'
+            }
+          />
+        </span>
+      )}
```

Comment:
We can refactor this to avoid duplication and use the internationalisation to something like:

```
type status = 'deaccessioned' | 'draft'

function CitationDatasetStatus({ status }: { status: status }) {
  const { t } = useTranslation('dataset')
  return (
    <span>
      t(dataset.status)
      <Tooltip placement={'top'} message={t(`${dataset.status}.description`)} />
    </span>
  )
}
```